contracts/core/mpl-rewards/v1/test/helper/Rewards.sol

```diff
@@ -128,22 +127,23 @@ contract CustodialTestHelper is TestUtil {
         assertEq(stakeToken.custodyAllowance(address(fez), address(custodian)), custodyAmt);
         assertEq(stakeToken.totalCustodyAllowance(address(fez)),                custodyAmt);
 
-        uint256 pullBackAmt = (amt - custodyAmt);
+        uint256 withdrawAmt = amt - custodyAmt;
 
         assertEq(stakeToken.balanceOf(address(fez)), amt);
 
         if (isPfdtStakeAsset) {
             make_withdrawable(fez, pool1);
 
-            assertTrue(!fez.try_withdraw(address(stakeToken), toUsd(pullBackAmt) + 1));
-            assertTrue( fez.try_withdraw(address(stakeToken),     toUsd(pullBackAmt)));
-            assertEq(  usdc.balanceOf(   address(fez)),             toUsd(pullBackAmt));
+            assertTrue(!fez.try_withdraw(address(stakeToken), toUsd(withdrawAmt) + 1));
+            assertTrue( fez.try_withdraw(address(stakeToken),     toUsd(withdrawAmt)));
+
+            assertEq(usdc.balanceOf(address(fez)), toUsd(withdrawAmt));
 
         } else {
             make_unstakeable(Staker(address(fez)), stakeLocker1);
 
-            assertTrue(!fez.try_unstake(address(stakeToken), pullBackAmt + 1));
-            assertTrue( fez.try_unstake(address(stakeToken),     pullBackAmt));
+            assertTrue(!fez.try_unstake(address(stakeToken), withdrawAmt + 1));
+            assertTrue( fez.try_unstake(address(stakeToken),     withdrawAmt));
```

Comment:
See original comment for alignment reference

Reply:
Regarding the rule, it should always be right aligned, unless there is an obvious reason to align to the left (lines with the same beginning). Its not a strict rule, more of a judgement call to improve readability.